de.metas.business/src/main/java/org/adempiere/impexp/product/ProductPriceImporter.java

```diff
@@ -61,17 +62,19 @@ public void createProductPrice_And_PriceListVersionIfNeeded()
 	{
 		if (!isValidPriceRecord())
 		{
-			return;
+			throw new AdempiereException("ProductPriceImporter.InvalidProductPriceList");
 		}
 
-		final I_M_PriceList_Version plv = getCreatePriceListVersion(request.getPriceListId(), request.getValidDate());
-		createProductPriceOrUpdateExistentOne(plv);
+		if (request.getPriceListId() > 0)
```

Comment:
i think that if we arrived here, if means that 
* `isValidPriceRecord()`  returned true which means that
* `request.getPriceListId() > 0`

so in summary, i don't understand what you are doing here :blush: ..or maybe you removed the wrong line in `isValidPriceRecord()` ?

Reply:
Yes, I removed the wrong line and then reconsidered..
This should be : request.getPrice().signum() > 0
Fixed!